Dockerfile

```diff
@@ -6,9 +6,14 @@ RUN apt-get update --yes \
     build-essential \
     less \
     vim \
+    curl \
   && gem install bundler:'~> 2.5' \
   && rm -rf /var/lib/apt/lists/*
 
+RUN curl -fsSL https://bun.sh/install | bash
+
+ENV PATH="/root/.bun/bin:${PATH}"
+
```

Comment:
Ensure bun is available in the development environment when ssh-ing into the `app` container.